index.html

```diff
@@ -2,13 +2,196 @@
+    </div>
+    <div class="header--content--main">
+    <div class="header--content--text">
+      <p class="header--content--text--paragraph">
+        Web Summit 2022 will bring together 70,000+ people, and the companies redefining the tech industry.
+      </p>
+    </div>
+    <div class="header--content-location">
+    <h2 class="header--content--time">Join us in Lisbon this November.</h2>
+    <p class="header--content--address">@ National Museum of Lisbon, ARt Center Nabi and more</p>
+    </div>
+    </div>
+
+  </div>
+
+  </header>
+
+  <main>
+    <section class="programs">
+      <div class="programFirst">
+      <h3 class="programs--title">Main Program</h3>
+      <div class="borderLine"></div>
+    </div>
+      <div class="programCards">
+      <div class="programCard">
+        <i class="fa-brands fa-speakap"></i>
+       <h4 class="programCard--title">Lecture</h4> 
+        <p class="programCard--text">
+          Listen to the speakers from various
+          countries about the messeges of 
+          sharing and opening.
+        </p>
+      </div>
+      <div class="programCard">
+        <i class="fa-solid fa-chalkboard-user"></i>
+        <h4 class="programCard--title">Exhbition</h4>
+        <p class="programCard--text">
+          As technology evolves, the human love of storytelling remains. How will the rise of at-home and on-demand content discovery change the way industries create?
+        </p>
+      </div>
+      <div class="programCard">
+        <i class="fa-solid fa-comments"></i>
+        <h4 class="programCard--title">Forum</h4>
+        <p class="programCard--text">
+          Get answers to the questions you’ve always wanted to ask on one of the most honest and open stages in tech. Join us in Lisbon this November.
+        </p>
+      </div>
+      <div class="programCard">
+        <i class="fa-solid fa-chalkboard"></i>
+        <h4 class="programCard--title">Workshop</h4>
+        <p class="programCard--text">
+          Take advice from founders, CEOs and investors at Startup University, and see how they got from where you are to where they are.
+        </p>
+      </div>
+      <div class="programCard">
+        <i class="fa-brands fa-connectdevelop"></i>
+        <h4 class="programCard--title">Ignite</h4>
+        <p class="programCard--text">
+          Remote is all about how we get the job done today. It looks at issues of work, including the continuing trend for remote working post pandemic.
+        </p>
+      </div>
+      </div>
+      <a href="#" class="programs--button">
+        Join CC Global Summit 2015
+      </a>
+    </section>
+    <section class="speakers">
+      <h3 class="speakers--title">Featured Speakers</h3>
+      <div class="borderLine"></div>
+      <div class="speakersGrid" id="speakersGrid">
+    </div>
+      <a href="#" class="speakers--button">
+        More 
+        <i class="fa-solid fa-chevron-down"></i>
+      </a>
```

Comment:
- [Optional] I don't think you need this **More** button again since you have decided to display all the speakers in the mobile section.